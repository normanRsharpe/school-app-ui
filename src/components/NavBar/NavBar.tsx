import React from 'react';

const NavBar = () => {
  return (
    <div className="w-100 overflow-hidden h3 bg-moon-gray flex items-center">
      <h1 className="f4 mid-gray center">APPLICATION</h1>
    </div>
  );
};

export default NavBar;
