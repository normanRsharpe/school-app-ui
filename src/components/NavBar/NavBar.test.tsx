import React from 'react';
import renderer from 'react-test-renderer';
import NavBar from '../../components/NavBar/NavBar';

describe('<NavBar>', () => {
  it('renders a default view', () => {
    const tree = renderer.create(<NavBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
