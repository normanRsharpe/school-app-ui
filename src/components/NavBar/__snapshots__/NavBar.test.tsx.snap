// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavBar> renders a default view 1`] = `
<div
  className="w-100 overflow-hidden h3 bg-moon-gray flex items-center"
>
  <div
    className="h2 w2 bg-mid-gray br-100 mh3"
  />
  <h1
    className="f4 mid-gray"
  >
    SQDN NAME
  </h1>
</div>
`;
